### DISCUSSION TITLE
SLA?

### DISCUSSION THREAD

**Discussion by thenewguy** [DrLex0/print3D-FlexiRex#8] [Q&A] [answered]
Just saw this for the first time and it looks fun. I have an SLA printer and I noticed the README mentions to use 15% infill or it doesn't balance properly. Any chance it will work well when printed on an SLA printer (pretty much 100% infill because of how it works)? Or would it have to be hollowed out in some fashion?

**Comment by DrLex0** [comment, marked as answer]
Well, I have seen some photos of SLA prints of the Flexi Rex standing upright, so it should be at least borderline stable when printed solid. However, I expect it to be very prone to face-planting at the slightest bump or when standing on a slightly downward slanted surface. I verified this by printing one with 100% infill. It is as I expected: it can stand upright, but it is very unstable.

The problem lies with the head. When printing the Rex with FFF at the recommended settings (3 perimeters, 15% infill), the head will have a much lower ratio of solid material versus empty space than the smaller parts thanks to its size. When increasing the infill, the head gains a lot more weight relative to the other parts, and this makes the model front-heavy.
So, to get a stable SLA print, you should hollow out the head in some way.
